code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -55,3 +55,25 @@
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	resistance_flags = NONE
 	armor = list("melee" = 15, "bullet" = 35, "laser" = 35, "energy" = 20, "bomb" = 35, "bio" = 35, "rad" = 35, "fire" = 0, "acid" = 0)
+
+/obj/item/clothing/gloves/rapid
+	name = "Gloves of the north star"
+	desc = "Just looking at these fills you with an urge to beat the shit out of people."
+	icon_state = "rapid"
+	item_state = "rapid"
+	transfer_prints = TRUE
+	var/warcry = "AT"
+
+/obj/item/clothing/gloves/rapid/Touch(mob/living/target,proximity = TRUE)
+	var/mob/living/M = loc
+
+	if(M.a_intent == INTENT_HARM)
+		M.changeNext_move(CLICK_CD_RAPID)
+		if(warcry)
+			M.say("[warcry]", ignore_spam = TRUE)

```

Comment:
The warcry sounds when clicking anywhere on-screen with harm intent, including machinery and areas out of vision, is this intended?

Reply:
It only does that when the touch action is applies which is when you click on something with an empty hand while wearing gloves.

Also yes very much intended as its funny, just the idea of someone yelling _ATATATATATATATATATATATAT!_ while typing on the HoP computer made me giggle in a way an adult man shouldn't.